LAYAIraq/LAYA-Iraq-Frontend
# Subtitles

Implementing #120 

# What to test
- make sure videos are shown (for vimeo, youtube and uploaded)
- make sure subtitles can be added for upload
